libsolidity/ast/AST.h

```diff
@@ -952,10 +958,6 @@ class VariableDeclaration: public Declaration, public StructurallyDocumented
 	bool isConstructorParameter() const;
 	/// @returns true iff this variable is a parameter(or return parameter of a library function
 	bool isLibraryFunctionParameter() const;
-	/// @returns true if the type of the variable does not need to be specified, i.e. it is declared
-	/// in the body of a function or modifier.
```

Comment:
Seems to be a comment for `canHaveAutoType()`, removed in 4c90ddf64ad22e2cae51d67269df3e326dbf4cb8.